### ISSUE TITLE
Re-design homepage

### ISSUE THREAD

**Issue by villanueval** [Smithsonian/Osprey#46]
The plotly graph doesn't meet accessibility requirements and haven't found other libs that do. Will re-design the homepage and add links to pages that can hold the details for each team and statistics for each project.

**Comment by villanueval** [comment]
Plotly lib removed and homepage is cards highlighting some projects, along with the main stats. 

- [ ] Add links to project stats

**Comment by villanueval** [comment]
Stats displayed in horizontal list.
